cherry_picker/cherry_picker.py

```diff
@@ -26,7 +26,7 @@ def cherry_pick(dry_run, pr_remote, commit_sha1, branches):
     if not branches:
         raise ValueError("at least one branch is required")
 
-    for branch in branches:
+    for branch in sorted(branches, reverse=True):
```

Comment:
This will already do the right thing for X.Y branches, but do we want to future-proof it against the day of 3.10 by adding `key=lambda v: tuple(map(int, v.split('.')))`?